### PR TITLE
I have added Hugo

### LINKED ISSUE #7: Add Hugo as a contributor to the project
closes #22

### PR DESCRIPTION
**Pull request by Hugogare**
closes #7

### CODE REVIEW

File: contributors.md [hunk 1]
```diff
@@ -2,4 +2,5 @@ Contributors file
 =================
 
 bananas24 - teacher
+Hugo - Student
```
Comment (bananas24) [inline contributors.md:5]: Hugo, I would like all dashes to be on the same column

**Review by bananas24 — CHANGES_REQUESTED** [review]

**Comment by JavierSff** [comment]
Solved the merging conflicts

**Review by LeireHernanG — APPROVED** [review]
Well done!

**Review by JavierSff — APPROVED** [review]
Good job!

**Review by JavierSff — APPROVED** [review]
Good job!

File: contributors.md (hunk 1 above)
Comment (Hugogare) [reply contributors.md]: changes are done

**Comment by bananas24** [comment]
Still missing the link to a PR

**Comment by Hugogare** [comment]
> Still missing the link to a PR

@bananas24 i have added the link to the pull request

**Review by bananas24 — APPROVED** [review]

**Pull request merged**